File: decompiled/sources/com/device/discovery/EcryptType.java
package com.device.discovery;

public enum EcryptType {
    ECRYPTTYPE_WE_NO(0),
    ECRYPTTYPE_WE_WEP(1),
    ECRYPTTYPE_WE_WPA(2);
    
    private int nCode;

    private EcryptType(int i) {
        this.nCode = i;
    }

    public String toString() {
        return String.valueOf(this.nCode);
    }
}

File: decompiled/sources/com/device/discovery/EcryptType.kt
package com.device.discovery

enum class EcryptType(private val nCode: Int) {
    ECRYPTTYPE_WE_NO(0),
    ECRYPTTYPE_WE_WEP(1),
    ECRYPTTYPE_WE_WPA(2);

    override fun toString(): String {
        return nCode.toString()
    }
}
